src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -80,6 +84,16 @@ const TableHeroImage = ({
         }
       </div>
     }
+    { !!showPvgnaGuide && pvgnaGuideInfo && heroName &&
```

Comment:
Sorry for late answer, I think it's not needed here.

I usually use double negation to make sure I have a boolean value (e.g. when the variable can be null.) for logical operators. But here it doesn't matter for the output.